scalableminds/webknossos
# In CI, Assert no Warnings From  Scala Compiler

Configures the scala compiler to fail on warnings, only in the CI.
I’m open for opinions, should we (1) do this at all and (2) do this locally as well? I personally think that CI-only is the best compromise, this way no code with compiler warnings should land in master, while local development is not hindered (unused variables are not unusual during development). I’m open for feedback :)

### Steps to test:
- CI run should fail if there are scala compiler warnings, e.g. unused local variables. Example run https://app.circleci.com/pipelines/github/scalableminds/webknossos/8054/workflows/4128fed4-5f08-4909-a5fd-e8e6e7c6f8d1/jobs/19412
- Ci should pass if there are no scala compiler warnings, example run https://app.circleci.com/pipelines/github/scalableminds/webknossos/8107/workflows/be2313f3-103a-4d88-803d-89a9c1807b0e/jobs/19465

------
